apps/web/src/i18n/locales/es/translation.json

```
@@ -72,21 +74,27 @@
     "confirm_password": "Confirmar contraseña",
     "reset_password": "Restablecer contraseña",
     "password_changed": "¡Contraseña cambiada!",
+    "password_change_error": "No se pudo cambiar su contraseña. Verifique que haya ingresado la información correcta.",
```

to match tone of other messages it  should be

```suggestion
    "password_change_error": "No se pudo cambiar tu contraseña. Verifica que hayas ingresado la información correctamente.",
```

---

Thanks!